lib/Drush/CommandFiles/ExampleCommandFile.php

```diff
@@ -46,9 +48,9 @@ public function exampleTable($options = ['format' => 'table', 'fields' => ''])
      * @option $french Add a row with French numbers.
      * @usage example:formatters --french
      */
-    public function alterFormatters($result, array $args, AnnotationData $annotationData)
+    public function alterFormatters($result, CommandData $commandData)
     {
-        if ($args['options']['french']) {
+        if ($commandData->input()->getOption('french')) {
```

Comment:
CommandData is only used in the hook system; there are no other clients of that, so I could bend the semver and change its name in 2.1.0 and it wouldn't hurt anything.

CommandData includes information about the command for a hook.  The information it includes is the Input object (which is where getOption() comes from -- Symfony's InputInterface::getOption()), the Output object, and the annotation data, which includes the command name, but does not include the entire command record.